package.json

```diff
@@ -25,5 +25,12 @@
     "colors": "^1.1.2",
     "dotenv": "^2.0.0",
     "express": "^4.14.0"
+  },
+  "devDependencies": {
+    "eslint": "^3.2.1",
+    "eslint-config-airbnb": "^9.0.1",
+    "eslint-plugin-import": "^1.12.0",
+    "eslint-plugin-jsx-a11y": "^2.0.1",
+    "eslint-plugin-react": "^5.2.2"
```

Comment:
Yeah, I tried uninstalling but eslint didn't like that. It seems that they're right now refactoring how their dependencies work, but until then we're stuck with importing all the plugins.